build/modules/fundingManager/FM_Rebasing_v1_config.json

```diff
@@ -50,16 +50,16 @@
     "EIP712_REVISION": { "tags": { "_0": [""] }, "returnNames": ["_0"] },
     "PERMIT_TYPEHASH": { "tags": { "_0": [""] }, "returnNames": ["_0"] },
     "allowance": {
-      "tags": { "owner_": [""], "spender": [""], "amount": ["decimals"] },
-      "returnNames": ["amount"]
```

Comment:
Are you aware of missing return names like here, and do we need to add them again? @mguleryuz

Reply:
No we dont need to replace them those are only for readability